### PR TITLE
remove error handler, and comment out specs

### PR DESCRIPTION
**Pull request by garcha**
https://github.com/caplinked/CpLnkd3/issues/5308

### CODE REVIEW

**Comment by jordanfowler** [comment]
CR'ed 👍

File: Gemfile.lock [hunk 1]
```diff
@@ -129,3 +129,6 @@ DEPENDENCIES
   simplecov
   vcr (~> 3.0.3)
   webmock (~> 2.1.0)
+
+BUNDLED WITH
```
Comment (aronslee) [inline Gemfile.lock]: remove 133 and 134

File: spec/core/client_spec.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  it "confirm default value initialize" do
+    client = Caplinked::Client.new
+
+    expect(client.api_host).to eq("sandbox.caplinked.com")
+    expect(client.api_scheme).to eq("https")
+  end
+
+  it "passing new values" do
+    client = Caplinked::Client.new api_key: ENV["API_KEY"], api_host: 'secure.caplinked.com', api_scheme: 'https'
```
Comment (aronslee) [inline spec/core/client_spec.rb]: Just use some dummy string for api_key instead of ENV var

File: spec/core/client_spec.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+    expect(client.api_host).to eq("secure.caplinked.com")
+    expect(client.api_scheme).to eq("https")
+    expect(client.api_key).to eq(ENV["API_KEY"])
```
Comment (aronslee) [inline spec/core/client_spec.rb]: Just use some dummy string for api_key instead of ENV var

File: spec/core/client_spec.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+RSpec.describe "Client", :type => :client do
+
+  it "confirm default value initialize" do
+    client = Caplinked::Client.new
+
+    expect(client.api_host).to eq("sandbox.caplinked.com")
+    expect(client.api_scheme).to eq("https")
```
Comment (aronslee) [inline spec/core/client_spec.rb:9]: should only accept https or http and default to https in client

File: spec/core/request.rb [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  client = Caplinked::Client.new api_key: "API_KEY", api_host: 'sandbox.caplinked.com', api_scheme: 'https'
+
+  it "execute init" do
+    stub_request(:get, "https://sandbox.caplinked.com/api/v1/files/38187?workspace_id=5886").
+      with(headers: {'Connection'=>'close', 'Host'=>'sandbox.caplinked.com', 'User-Agent'=>'http.rb/2.1.0', 'X-Token'=> "API_KEY"}).
+      to_return(status: 200, body: fixture('activities_workspace.json'), headers: { content_type: 'application/json; charset=utf-8' })
+
+    request = Caplinked::Request.new(client, :get, "/api/v1/activities/workspace/5886", {:params=>{}})
+
+    expect(request.client.api_host).to eq("sandbox.caplinked.com")
```
Comment (aronslee) [inline spec/core/request.rb:9]: would be good to also test result of @uri

File: spec/core/request.rb [hunk 6]
```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+RSpec.describe "Request", :type => :request do
+  client = Caplinked::Client.new api_key: "API_KEY", api_host: 'sandbox.caplinked.com', api_scheme: 'https'
+
+  it "execute init" do
+    stub_request(:get, "https://sandbox.caplinked.com/api/v1/files/38187?workspace_id=5886").
```
Comment (aronslee) [inline spec/core/request.rb]: you dont need this stub here right bc you are not calling perform on the http request?

**Review by aronslee — CHANGES_REQUESTED** [review]

**Comment by aronslee** [comment]
@garcha also test & fixture json response for a couple error examples would nice

File: spec/core/client_spec.rb (hunk 4 above)
Comment (garcha) [reply spec/core/client_spec.rb:9]: default is https, but any value can be passed in. I think its fine, the users that will have access are paying customers anyways.

File: spec/core/client_spec.rb (hunk 2 above)
Comment (garcha) [reply spec/core/client_spec.rb]: done

File: spec/core/client_spec.rb (hunk 3 above)
Comment (garcha) [reply spec/core/client_spec.rb]: done

File: spec/core/request.rb (hunk 6 above)
Comment (garcha) [reply spec/core/request.rb]: done

File: spec/core/request.rb (hunk 5 above)
Comment (garcha) [reply spec/core/request.rb:9]: done

**Review by aronslee — APPROVED** [review]

**Comment by garcha** [comment]
Ready to merge.

**Pull request merged**